authorization/permission/model/permission_model_service.go

```diff
@@ -0,0 +1,179 @@
+package model
+
+import (
+	"context"
+
+	"github.com/fabric8-services/fabric8-auth/authorization/repository"
+	"github.com/jinzhu/gorm"
+	uuid "github.com/satori/go.uuid"
```

Comment:
we can simply write `"github.com/satori/go.uuid"` - we have this inconsistency in a number of places (outside the PR 
) though.

Reply:
Fixed